### PR TITLE
Rename ansible builtin calls using fully qualified names

### PR DESCRIPTION
**Pull request by fmount**
As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date:
- [x] README in the role
- [x] Content of the docs/source is reflecting the changes

### CODE REVIEW

File: ci_framework/roles/cifmw_cephadm/tasks/pre.yml
Comment (cjeanner) [inline ci_framework/roles/cifmw_cephadm/tasks/pre.yml:1]: `ansible.builtin.setup` and `ansible.builtin.package` are missing here :)

File: ci_framework/roles/cifmw_cephadm/tasks/selinux.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@
   block:
     - name: Set fcontext on ceph config and admin keyring
       become: true
-      sefcontext:
+      ansible.builtin.sefcontext:
```
Comment (cjeanner) [inline ci_framework/roles/cifmw_cephadm/tasks/selinux.yml:21]: that's `community.general.sefcontext`:
https://docs.ansible.com/ansible/latest/collections/community/general/sefcontext_module.html

File: ci_framework/roles/cifmw_cephadm/tasks/pre.yml
Comment (fmount) [reply ci_framework/roles/cifmw_cephadm/tasks/pre.yml:1]: yeah missed them from my `sed` :D

File: ci_framework/roles/cifmw_cephadm/tasks/monitor_ceph_orch.yml
Comment (cjeanner) [inline ci_framework/roles/cifmw_cephadm/tasks/monitor_ceph_orch.yml:1]: missing `ansible.builtin.set_fact` on line 40

File: ci_framework/roles/cifmw_cephadm/molecule/default/tasks/verify.yml
Comment (cjeanner) [inline ci_framework/roles/cifmw_cephadm/molecule/default/tasks/verify.yml:1]: Missing `ansible.builtin.include_vars` on line 61

File: ci_framework/roles/cifmw_cephadm/molecule/default/converge.yml
Comment (cjeanner) [inline ci_framework/roles/cifmw_cephadm/molecule/default/converge.yml:1]: Missing `ansible.builtin.shell` on line 37

File: ci_framework/roles/cifmw_ceph_client/tasks/main.yml
Comment (cjeanner) [inline ci_framework/roles/cifmw_ceph_client/tasks/main.yml:1]: Missing `ansible.builtin.include_vars` on line 24

**Review by cjeanner — COMMENTED** [review]
I think I caught them all :). Just a couple of things, but that's close to a win!

File: ci_framework/roles/cifmw_cephadm/tasks/monitor_ceph_orch.yml
Comment (fmount) [reply ci_framework/roles/cifmw_cephadm/tasks/monitor_ceph_orch.yml:1]: Done

File: ci_framework/roles/cifmw_cephadm/molecule/default/converge.yml
Comment (fmount) [reply ci_framework/roles/cifmw_cephadm/molecule/default/converge.yml:1]: Done

File: ci_framework/roles/cifmw_ceph_client/tasks/main.yml
Comment (fmount) [reply ci_framework/roles/cifmw_ceph_client/tasks/main.yml:1]: Done

Comment (fmount) [reply ci_framework/roles/cifmw_ceph_client/tasks/main.yml:1]: Done

File: ci_framework/roles/cifmw_cephadm/molecule/default/converge.yml
Comment (fmount) [reply ci_framework/roles/cifmw_cephadm/molecule/default/converge.yml:1]: Done

File: ci_framework/roles/cifmw_cephadm/molecule/default/tasks/verify.yml
Comment (fmount) [reply ci_framework/roles/cifmw_cephadm/molecule/default/tasks/verify.yml:1]: Done

File: ci_framework/roles/cifmw_cephadm/tasks/monitor_ceph_orch.yml
Comment (fmount) [reply ci_framework/roles/cifmw_cephadm/tasks/monitor_ceph_orch.yml:1]: Done

**Review by cjeanner — COMMENTED** [review]
/approve

that's a huge change, but necessary. As a follow-up, I'll ensure the pre-commit "ansible-lint" task actually DOES lint things up... seems it didn't catch those issues beforehand :(.

**Pull request merged**